scripts/plugins/storage/sqlite/upgrade/5.sql

```diff
@@ -1,8 +1,16 @@
-UPDATE foglamp.configuration SET value = '{"plugin": {"description": "OMF North Plugin", "type": "string", "default": "omf", "value": "omf"}, "source": {"description": "Source of data to be sent on the stream. May be either readings, statistics or audit.", "type": "string", "default": "audit", "value": "readings"}}'
+UPDATE foglamp.configuration SET value = json_set(value, '$.stream_id', json('{"type":"integer","description":"Stream ID","default":"1","value":"1"}'))
+       WHERE key = 'North Readings to PI';
+UPDATE foglamp.configuration SET value = json_set(value, '$.source', json('{"plugin": {"description": "OMF North Plugin", "type": "string", "default": "omf", "value": "omf"}, "source": {"description": "Source of data to be sent on the stream. May be either readings, statistics or audit.", "type": "string", "default": "audit", "value": "readings"}}'))
```

Comment:
I think this should use be

UPDATE foglamp.configuration SET value = json_set(value, '$.source', json('{"source": {"description": "Source of data to be sent on the stream. May be either readings, statistics or audit.", "type": "string", "default": "audit", "value": "readings"}}'))

The plugin should already be there and be correct.

Reply:
Yes, oversight. Fixed